recipes/r-justifyalpha/meta.yaml

```diff
@@ -0,0 +1,94 @@
+{% set version = '0.1.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-justifyalpha
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/JustifyAlpha_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/JustifyAlpha/JustifyAlpha_{{ version }}.tar.gz
+  sha256: c929630414aa79e1adf9a32b57f254b086a91d3a3874810b55b324ab89d5a72d
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: knitr, rmarkdown, testthat,
```

Comment:
This can be removed.

Reply:
Just the comment right? Still need `lib/R/lib` and `lib/`?

Reply:
✔️